spec/lib/administrate/search_spec.rb

```diff
@@ -10,6 +11,8 @@ class MockDashboard
     name: Administrate::Field::String,
     email: Administrate::Field::Email,
     phone: Administrate::Field::Number,
+    status: Administrate::Field::Enum.with_options(
+      enum: {'pending' => 0, 'confirmed' => 1}),
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.